docs/worker-node/install-singularity.md

```diff
@@ -99,8 +99,28 @@ To install singularity as `setuid`, make sure that your host is up to date befor
 
 ### Configuring singularity ###
 
-There are no required changes to the default configuration.  If you want
-to see what options are available, see `/etc/singularity/singularity.conf`.
+The OSG distribution of singularity includes a feature called
+`underlay` that is not enabled by default but recommended because it
+is less vulnerable to security problems than the default `overlay`
+feature.
```

Comment:
Can you split this into two sentences? The first explaining what underlay is and the second explaining that it is not enabled by default but is recommended?